molecule/shared/cleanup.yml

```diff
@@ -0,0 +1,19 @@
+- name: unregister systems
+  hosts: all
+  gather_facts: false
+  tasks:
+    - block:
+        - name: wait for host
+          wait_for_connection:
+            timeout: 1
```

Comment:
I don't know if it's worth commenting on this in the playbook, but the timeout of 1 actually ends up being the value of `connect_timeout`, which defaults to 5. Since `timeout` is 1, this task ends up waiting one `connect_timeout` interval before checking to see if the max wait time exceeds the overall `timeout` value of 1, which it almost certainly will if the connection isn't established.

This was my hacky way of trying to express "only try to connect once".

Reply:
I personally don't feel the need to go into that level of detail. I feel like it's pretty clear what we're doing here, but maybe that's just because I wrote it and it's derived from you. What say you, @robled, @st-murray, @andrew103? Should this use more documentation or not?

Reply:
I'm in favor of "not". If any comment is needed, it could/should be along the lines of "may wait more than 1 second. this is fine."

Reply:
I don't think its too big of an issue. If anything, a shorty comment like Sean suggested would be fine.